packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -57,34 +57,31 @@ describe("IModelTransformerHub", () => {
   });
   after(() => HubMock.shutdown());
 
+  const createPopulatedIModelHubIModel = async (iModelName: string, prepareIModel: (iModel: SnapshotDb) => Promise<void>): Promise<string> => {
```

Comment:
```suggestion
  const createPopulatedIModelHubIModel = async (
    iModelName: string,
    prepareIModel?: (iModel: SnapshotDb) => void | Promise<void>
  ): Promise<string> => {
```
make second argument optionally async and in general optional

Reply:
Changed it